CHANGELOG.md

```diff
@@ -8,10 +8,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Add support for Amazon Voice Focus support in Safari Technology Preview for macOS.
```

Comment:
maybe add which version of the technology review

Reply:
That's in the support table, at least.